<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fieldWindow</class>
 <widget class="QDialog" name="fieldWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Champs de formulaire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Champs du formulaire</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelLabel">
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="labelLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="texteAideLabel">
           <property name="text">
            <string>Texte d'aide</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="texteDAideLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="typeLabel">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="typeComboBox">
           <item>
            <property name="text">
             <string>Texte</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nombre</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Email</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>URL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Heure</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date et heure</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Choix multiple</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="placeholderLabel">
           <property name="text">
            <string>Placeholder</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="placeholderLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="valeurParDefautLabel">
           <property name="text">
            <string>Valeur par défaut</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="valeurParDefautLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="requisLabel">
           <property name="text">
            <string>Requis</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="requisCheckBox"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="choixMultipleLabel">
           <property name="text">
            <string>Réponses multiples</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QCheckBox" name="choixMultipleCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Contraintes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollConstraintsArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaConstraintsContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>290</width>
              <height>168</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addConstraintButton">
             <property name="text">
              <string>Ajouter contrainte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fieldWindow</receiver>
   <slot>valid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addConstraintButton</sender>
   <signal>clicked()</signal>
   <receiver>fieldWindow</receiver>
   <slot>addConstraint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>fieldWindow</receiver>
   <slot>selectChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valid()</slot>
  <slot>addConstraint()</slot>
  <slot>selectChange(int)</slot>
 </slots>
</ui>
